app/helpers/application_helper.rb

```diff
@@ -13,4 +13,9 @@ def markdown_to_html(markdown)
     redcarpet = Redcarpet::Markdown.new(renderer, extensions)
     (redcarpet.render markdown).html_safe
   end
+
+  def sortable(column, title = nil)
+    title ||= column
+    link_to title, {:sort => column}
```

Comment:
To add a `sorter` CSS class to every sortable `<a>` tag, modify line 19 to:
```ruby
link_to title, {:sort => column}, class: 'sorter'
```
Look at the table header links this generates, they will look like:

```html
<th><a href="..." class="sorter"></th>
```
The `.sorter` CSS class can be used to style the sorter links, e.g. if you wanted to add a 10px red border to them with giant text:
```css
a.sorter {
  border: 10px solid red;
  font-size: 5em;
}
```